csharp/src/Microsoft.ML.OnnxRuntime/PinnedOnnxValue.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.ML.OnnxRuntime.Tensors;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    /// <summary>
+    /// Represents an Onnx Value with its underlying buffer pinned
+    /// </summary>
+    public class PinnedOnnxValue : IDisposable
+    {
```

Comment:
Can this be marked as internal instead of public? If external users don't need to use it directly, should limit it to internal.

Reply:
looks like constructor is internal